budget/budgetrouter.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+
+const router = express.Router();
+const Budget = require('./budget.js')
+
+router
+.route('/').post((req, res) => {
+    
+     const { body } = req;
+    const budget = new Budget(body)
+    
+    budget.save()
+        .then(budgets => {
+            res.status(200).json(budgets)
```

Comment:
HTTP status code should be `201` since we're creating a new document and send it to our client (Postman).

https://httpstatuses.com/201